renderdoc/driver/gl/gl_driver.h

```diff
@@ -279,6 +279,11 @@ class WrappedOpenGL : public IFrameCapturer
     // bound to the second.
     GLuint renderbufferReadTex;
     GLuint renderbufferFBOs[2];
+
+    // since compressed textures cannot be read back on GLES we have to store them during the
+    // uploading
+    // target -> level -> data
+    map<int, map<int, vector<byte> > > compressedData;
```

Comment:
I think it's needed because of cube faces. However they could be stored successively, but that would require some modifications on the current implementation.